main.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require "report_builder"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mpermperpisang/golang-automation/blob/4c37372fb449e752801dcb55d55b269bb510a466/main.rb#L3"></span>Prefer single-quoted strings when you don’t need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>